hca/dss/util/__init__.py

```diff
@@ -58,7 +58,13 @@ def hardlink(source, link_name):
         if source_stat.st_dev != dest_stat.st_dev or source_stat.st_ino != dest_stat.st_ino:
             raise
     except OSError as e:
-        if e.errno == errno.EMLINK:
+        # See https://docs.python.org/3/library/errno.html
+        copy_on_error = [
+            errno.EMLINK,  # too many links
+            errno.EACCES,  # permission denied
+            errno.EPERM,  # operation not permitted
```

Comment:
```suggestion
            errno.EMLINK,
            errno.EACCES,
            errno.EPERM,
```

The meaning of the constants is [well documented](https://docs.python.org/3.6/library/errno.html#errno.EMLINK). We don't need to duplicate that here. What isn't obvious is the intent e.g., *why* EPERM is included. Like that we expect (and have observed) this to occur on an NFS share.

Reply:
Comments removed, better descriptive (why EPERM or EMLINK) added